doc/changes/changes_0.1.1.md

```diff
@@ -12,3 +16,10 @@
 * #15: Added a link to the development guide.
 
 ## Dependency updates
+
+* Added `com.fasterxml.jackson.core:jackson-dataformat-cbor:2.12.0`
+* Updated `com.fasterxml.jackson.core:jackson-databind:2.11.2` to `2.12.0`
```

Comment:
Use latest aggreed format. 
```
Updated to `artifact:version` (was `previous-version`)
```

Reply:
When was it changed? :O I thought we agreed on this one and this is a template: https://github.com/exasol/release-droid/blob/main/doc/user_guide/changes_file_template.md

Reply:
I like the old format more...

Reply:
I think we misunderstood, this is should be default.